recipes/tiny-aes-c/all/conanfile.py

```diff
@@ -0,0 +1,100 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.32.0"
+
+class TinyAesCConan(ConanFile):
+    name = "tiny-aes-c"
+    license = "Unlicense"
+    homepage = "https://github.com/kokke/tiny-AES-c"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Small portable AES128/192/256 in C"
+    topics = ("encryption", "crypto", "AES")
+
+    settings = "os", "compiler", "build_type", "arch"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        # AES128, AES192 or AES256
+        "aes_block_size": ["AES128", "AES192", "AES256"],
+        # enable AES encryption in CBC-mode of operation
+        "cbc": [True, False],
+        # enable the basic ECB 16-byte block algorithm
+        "ecb": [True, False],
+        # enable encryption in counter-mode
+        "ctr": [True, False],
```

Comment:
these information must be injected at build time.

Reply:
https://github.com/kokke/tiny-AES-c/blob/7f70d54d6e8e22dec58a59254f53c3e66e978199/aes.c#L338

Reply:
also: https://github.com/kokke/tiny-AES-c/blob/7f70d54d6e8e22dec58a59254f53c3e66e978199/aes.c#L47-L56